### PR TITLE
Update zh-CN Translation

### PR DESCRIPTION
**Pull request by xbmanh**

### CODE REVIEW

**Comment by xbmanh** [comment]
I Fixed some bugs and added some translations.

File: _config.yml [hunk 1]
```diff
@@ -6,16 +6,16 @@ baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://12z1.com"
 github_username:  EKA2L1
 
-languages: [ "en", "vi", "es", "pt-br", "zh-cn", "ru" ]
+languages: [ "en", "zh-cn", "vi", "es", "pt-br", "ru" ]
```
Comment (pent0) [inline _config.yml]: I'm very thankful for you listening to me, but I'm not really sure if this change is ill-intended. Very sorry, you could have done it in the beginning but now it just looks weird. I will hear explanation, I will try to understand, thanks

Comment (xbmanh) [reply _config.yml]: I want to put zh-CN option in front of English on the language menu.

**Comment by xbmanh** [comment]
I want to move the zh-CN option in front of vi on language menu.

**Comment by xbmanh** [comment]
我这么做可以把简体中文选项放在英语后面吗?

File: _config.yml (hunk 1 above)
Comment (xbmanh) [reply _config.yml]: 我这么做可以把简体中文选项放在英语后面吗?

Comment (pent0) [reply _config.yml]: 如果你要，这个好，但你有理由吗？我不想把政治牵扯到这件事情上，因为我是越南

Comment (pent0) [reply _config.yml]: 你可以从头开始做哈哈

Comment (xbmanh) [reply _config.yml]: 没有牵涉政治

Comment (pent0) [reply _config.yml]: 对不起， 我没有恶意

In the future, I will adjust the language to other website orders (like Steam for example). I don't mean any offense

**Pull request merged**